docs/milestone2.md

```diff
@@ -176,77 +176,33 @@ We will use [`setuptools`](https://packaging.python.org/tutorials/packaging-proj
 
 ## Implementation details
 
-## TODO
+### Current Implementation
 
-### Data Structures
-- scalar(integers or floats)
-- dictionary(can be used to initialze/assign variables)
-	
-	e.g. we can initiate input values by calling: `initial_condition({x:1, y:2})`
-- vector
+#### Core Data Structures: 
+- tensor
+- dictionary
+- vector(to be implemented)
```

Comment:
I don't think we'll have `pyad.vector` since we should be able to encompass that with `pyad.Tensor(np.array([1, 2, 3])` or something like that.

So, I think that this section could probably get merged with the "Core Classes" section. Or rather, the "Core Classes" as it is right now could be renamed to "Core Data Structures"